### DISCUSSION TITLE
Dataverse action response parsing

### DISCUSSION THREAD

**Discussion by eswarpr** [microsoft/Power-Fx#1998] [Q&A]
Wondering if someone can help answer me on this one.

Let's say I have an unbound action in Dataverse which I need to call from a Canvas App. It accepts open type entity (expando) and returns an open type entity (expando).

I have figured out we can use ParseJSON and JSON functions to create the input parameter for it - which works well. But how do I parse the return value in PowerFX?

I can't figure it out. The app studio says the result is a Record but without any attributes from the result?

**Comment by jorisdg** [comment]
Have you checked out the article "Working with JSON"? https://learn.microsoft.com/power-platform/power-fx/working-with-json

We currently do not have support for dynamically accessing or iterating over JSON properties, although it's something we definitely want to support in the Power Fx language in the future.
